src/deprecated.jl

```diff
@@ -22,3 +22,6 @@ import Base: keys, values, insert!
 
 @deprecate stackdf stackdt
 @deprecate meltdf meltdt
+
+@deprecate readtable CSV.read
```

Comment:
Do the signatures match? That is, if you say `readtable(x, y, z)`, this will tell you to do `CSV.read(x, y, z)`. Will that work?